besu/src/main/java/org/hyperledger/besu/cli/options/storage/DataStorageOptions.java

```diff
@@ -76,6 +79,7 @@ public static DataStorageOptions create() {
    */
   public void validate(final CommandLine commandLine) {
     diffBasedSubStorageOptions.validate(commandLine, dataStorageFormat);
+    verkleSubStorageOptions.validate(commandLine, dataStorageFormat);
```

Comment:
It's just to ensure that all the logic is coded in the same way so that if tomorrow someone adds a flag that requires a check, they don’t have to wonder why it doesn’t work or how to do it. They just need to follow the same approach as in other parts of the code and simply implement the check in `validate`. But if you think it's better to remove why not

Reply:
no strong opinions, you can leave it